misk/web/tabs/config/src/index.tsx

```diff
@@ -10,13 +10,23 @@ import App from "./App"
 import rootReducer from "./reducers"
 import rootSaga from "./sagas"
 
-// export * from "./components"
+export * from "./components"
 export * from "./containers"
 
-const history = createBrowserHistory()
+export interface IWindow extends Window {
+  __REDUX_DEVTOOLS_EXTENSION_COMPOSE__: any
+  Misk: {
+    Binder: any
+    History: any
+  }
+}
+const Window = window as IWindow
+
+Window.Misk.History = Window.Misk.History || createBrowserHistory()
```

Comment:
I wish we could pass / provide this to the sub components instead.

Reply:
Would definitely be preferable once we find a way to render them without compile time access to the tab components.